jwst/fits_generator/create_dms_data.py

```diff
@@ -111,7 +111,10 @@ def get_subarray_name(instrument, colstart, colstop, rowstart, rowstop):
         if rowstart == subarray[2] and rowstop == subarray[3] and colstart == subarray[0] \
                 and colstop == subarray[1]:
             return subarray[4]
-    return 'UNKNOWN'
+    #
+    # If we can't match the subarray parameters we return 'GENERIC', which is
+    # a legal subarray name
+    return 'GENERIC'
```

Comment:
Yes, it's a legal value, but technically only for reference files, never for science products. But what else can you do. It'll probably still cause errors when running the pipeline and calling bestref to find ref files, just like UNKNOWN. But at least it passes schema validation.

Reply:
The schema validation was what I was aiming for